<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CajaGiratoria</class>
 <widget class="QDialog" name="CajaGiratoria">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CajaGiratoria</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="rotuloNombre">
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editorNombre"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="rotuloEdad">
       <property name="text">
        <string>Edad</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="cajaGiraDeEdad"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="rotuloGrado">
       <property name="text">
        <string>Grado</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="cajaGiraDeGrado"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btnCaja">
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
